tools/cli/src/args.rs

```diff
@@ -50,17 +50,29 @@ pub enum Command {
 
 #[derive(Clone, Args)]
 pub struct TaskQueueArgs {
+    /// Task queue address. If not provided, then it will be read
+    /// from the environment variable LOCAL_TASK_QUEUE_ADDRESS or TEST_TASK_QUEUE_ADDRESS
```

Comment:
okay, cli flag then fallback to env.

Good for now. I think this could use some reflection to make sure everything is clear to users (after this sprint is over and there is breathing room)